test/specs/parser/tests/01.parser.js

```diff
@@ -455,6 +471,54 @@ exports.execute = function(options) {
                     options.ermRest.parse(baseUri + "/*::facets::invalidblob");
                 }).toThrow(facetError);
             });
+
+            it("should throw an error for invalid facet filter errors.", function() {
+                try{
+                      options.ermRest.parse(baseUri + "/*::facets::invalidblob@sort()");
+                } catch(e){
+                  expect(e).toEqual(jasmine.any(Object));
```

Comment:
These `expect(e).toEqual(jasmine.any(Object));`s are not necessary.

Reply:
Also if this doesn't throw any errors, the `it` will succeed.  Add a dummy `expect(false).toBe(true, "didn't throw any errors".)` after the parse inside the try-catch.